hail_search/queries/base.py

```diff
@@ -89,9 +89,12 @@ class BaseHailTableQuery(object):
 
     @classmethod
     def load_globals(cls):
-        ht_path = cls._get_table_path('annotations.ht')
-        ht_globals = hl.eval(hl.read_table(ht_path).globals.select(*cls.GLOBALS))
-        cls.LOADED_GLOBALS = {k: ht_globals[k] for k in cls.GLOBALS}
+        try:
```

Comment:
do we need to call this function again once the pipeline runs and annotations table exists / or will it be called again when the hail search service restarts after the pipeline runs?

Reply:
In our environment this isn't an issue though.

Reply:
I don't feel great about putting this in prod - `In our environment this isn't an issue though` is not necessarily true, as in the past if there was a problem with how the disk mounted or if the data got corrupted somehow the pod start up would fail and kubernetes would either make a new pod or at least it would not take down the old pods until there were successful new pods. Now it might silently fail to read any data but the pod would start up and the old pods would be deleted and then search would be down